lib/ronin/recon/cli/commands/run.rb

```diff
@@ -261,9 +266,9 @@ def run(*values)
 
             values = values.map { |value| parse_value(value) }
 
-            output_file = if options[:output] && options[:output_format]
-                            options[:output_format].open(options[:output])
-                          end
+            output_files = outputs.filter_map do |output, output_format|
+              output_format.open(output) if output && output_format
```

Comment:
The `output && output_format` test might not be necessary. Double checking `OutputFormats.infer_from`, it appears to default to `OutputFormats::TXT`, so we should always have an output format.

```ruby
Ronin::Recon::OutputFormats.infer_from('foo.lol')
# => Ronin::Core::OutputFormats::TXT
```

Reply:
There should always be an `--output` value if `--output` is given, since the argument is required.